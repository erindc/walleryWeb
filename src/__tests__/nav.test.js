import NavBar from '../shared/Nav';
import React from 'react'
import { shallow } from 'enzyme'

describe('NavBar', () => {
  it('renders the correct elements for authenticated', () => {
    const component = shallow(<NavBar authenticated={true} currentUser={'test'} />)
    expect(component).toMatchSnapshot()
  })

  it('renders the correct elements for unauthenticated', () => {
    const component = shallow(<NavBar authenticated={false} currentUser={'test'} />)
    expect(component).toMatchSnapshot()
  })
})