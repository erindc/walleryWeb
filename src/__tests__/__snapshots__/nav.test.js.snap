// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders the correct elements for authenticated 1`] = `
<Route>
  <Component />
</Route>
`;

exports[`NavBar renders the correct elements for unauthenticated 1`] = `
<Route>
  <Component />
</Route>
`;
